src/interface_r/DESCRIPTION

```diff
@@ -0,0 +1,29 @@
+Package: h2o4gpu
+Type: Package
+Title: h2o4gpu in R
+Version: 0.0.1.9999
```

Comment:
Let's use [Hadley's recommendation](http://r-pkgs.had.co.nz/description.html) of starting the package at 0.0.0.9000 and then bump the last digit every time someone adds something new (e.g. 0.0.0.9001, 0.0.0.9002, etc).  When all the algos are added, then we can take this out of development mode.

Reply:
Sounds good. Will change.